plugins/source/aws/client/account.go

```diff
@@ -78,30 +77,14 @@ func (c *Client) setupAWSAccount(ctx context.Context, logger zerolog.Logger, aws
 	if err != nil {
 		return nil, err
 	}
-	svcsDetails := make([]svcsDetail, len(account.Regions)+1)
-	for i, region := range account.Regions {
-		svcsDetails[i] = svcsDetail{
-			partition: iamArn.Partition,
-			accountId: *output.Account,
-			region:    region,
-			svcs:      initServices(region, awsCfg),
-		}
-	}
 
-	var cloudfrontRegion string
-	switch iamArn.Partition {
-	case "aws":
-		cloudfrontRegion = awsCloudfrontScopeRegion
-	case "aws-cn":
-		cloudfrontRegion = awsCnCloudfrontScopeRegion
-	}
-
-	svcsDetails[len(account.Regions)] = svcsDetail{
+	svcsDetails := svcsDetail{
 		partition: iamArn.Partition,
 		accountId: *output.Account,
-		svcs:      initServices(cloudfrontRegion, awsCfg),
+		svcs:      initServices(awsCfg, account.Regions),
 	}
-	return svcsDetails, nil
```

Comment:
This is not needed anymore because we add the appropriate client in the multiplexer without having to have the region in the list